README.md

```diff
@@ -46,7 +46,7 @@ Discovery.rx.service(configurations: [ configuration ] ).subscribe(onNext: { res
 Add the following entry to your package's dependencies:
 
```

Comment:
The section "## Requirements" above also needs to be updated

Reply:
fixed